### ISSUE TITLE
Super is not working

### ISSUE THREAD

**Issue by rares9301** [prasanthrangan/hyprdots#175]
Just installed and the task bar works fine. I can switch between themes, modify sound settings, view core/memory load. As the title says, i cant open anything else that is not in the taskbar. No commands that use SuperKey work. Also when i click the Packages are up to date button, it tries to open something but closes instantly

**Comment by rares9301** [comment]
Date and time is also not working, cant fix it since i dont see a way to open the terminal

**Comment by edanbigw** [comment]
> 

did u setup your timedatectl correctly?

**Comment by rares9301** [comment]
i did not since i used a base arch install with grub. Cant open terminal since SuperKey is not working :P

**Comment by prasanthrangan** [comment]
Is your system locale set correctly during arch install? if not, please switch to TTY1 `Ctrl+Alt+F1` and set your [locale](https://wiki.archlinux.org/title/locale) and system time.
Also please update keyboard layout in hyprland.conf and in [keybindings.conf](https://github.com/prasanthrangan/hyprdots/blob/79807bdfdfb26f94d342ed7b5312754ceccee961/Configs/.config/hypr/keybindings.conf#L10C31-L10C31)

**Comment by rares9301** [comment]
in keybindings.conf,  Mainmod was set to "SUPER # windows key". Changed to "CONTROL" and it now works. I synchronized the clock woth timedatectl and it also works now

TLDR: in keybindings.conf, "SUPER # windows key" had probably wrong syntax. Read something on the internet and the command keys are written using capitals. 🤷‍♂️
